components/patient-data/ui/src/main/resources/PhenoTips/GeneClass.xml

```diff
@@ -114,7 +174,7 @@
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <values>candidate=Candidate|rejected_candidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Negative</values>
+      <values>candidate=Candidate|candidate_novel_disease=Novel disease gene|candidate_novel_phen=Known disease gene with novel phenotype|umc=Uncertain molecular cause|umc_vus=Known disease gene with a variant of unknown significance|umc_msv=Known disease gene missing a second variant|rejectedcandidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Tested negative</values>
```

Comment:
To circumvent the need for renaming and migrating existing status values, replace the character that marks sub-statuses:
```suggestion
      <values>candidate=Candidate|candidate>novel_disease=Novel disease gene|candidate>novel_phen=Known disease gene with novel phenotype|umc=Uncertain molecular cause|umc>vus=Known disease gene with a variant of unknown significance|umc>msv=Known disease gene missing a second variant|rejected_candidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Tested negative</values>
```